src/wmnds/components/disruption-indicator/medium/_medium.njk

```diff
@@ -21,7 +23,15 @@
     })
   }}
   {% endif %}
-  {{ _content }}
+  {# set content #}
+  {% if _content %}
+    <span>{{ _content }}</span>
+  {% else %}
+  {# set default content if there is no iconLeft #}
+    {% if hasIconLeft !== true %}
+    <span>X15</span>
```

Comment:
Yeah its just because the css checks for a sibling element to decide spacing between the icons and plain text isn't considered an element. This seemed like the quickest/easiest fix but you may think of a better one

Reply:
@houbly  No worries, I've updated it and gave the span a class "wmnds-disruption-indicator-medium__service" so we can easily style it in future.